pkgs/by-name/as/astroterm/package.nix

```
@@ -0,0 +1,72 @@
+{
+  stdenv,
+  lib,
+  fetchurl,
+  fetchFromGitHub,
+  xxd,
+  meson,
+  ninja,
+  ncurses,
+  argtable,
+}:
+
+stdenv.mkDerivation rec {
```

```suggestion
stdenv.mkDerivation (finalAttrs: {
```
Not blocking, but it'd be great to reduce the number of `rec`s in Nixpkgs!